src/olympia/abuse/cinder.py

```diff
@@ -353,7 +353,7 @@ def get_attributes(self):
             'name': self.get_str(self.addon.name),
             'slug': self.addon.slug,
             'summary': self.get_str(self.addon.summary),
-            'promoted': self.get_str(promoted_group.name),
+            'promoted': self.get_str(promoted_groups.get_names(needs_badged=False)),
```

Comment:
If `.get_names` is just filtering by badged or not badged and then returning the .name is it really beneficial to have this as a model method at all? How is it different/better than the shorter and built in?

```python
promoted_group.filter(badged=False).name
```

Reply:
I was concerned with the future use of `.name` and the possibility of something similar to what exists already occurring again (since its not immediately obvious that public names should be filtered out.) I do prefer this way and it's what I did initially, though. Given there's only two places where filtering is necessary, probably just better to go with this.